app/views/course/assessment/assessments/index.html.slim

```diff
@@ -1,9 +1,9 @@
 = page_header format_inline_text(@category.title) do
   = new_button([current_course, :assessment]) if can?(:create, Course::Assessment.new(tab: @tab))
-
-= tabs do
-  - @category.tabs.each do |item|
-    = nav_to(format_inline_text(item.title),
+- if has_more_than_one_tab?
+  = tabs do
```

Comment:
Maybe I'm not clear enough.

I think the entire construction of tabs should be moved to the helper, so the helper is in charge of rendering the tabs (or returning nothing if there are no tabs)

This allows you to directly check the generated output for tabs using HTML matchers instead.